### DISCUSSION TITLE
hit notes in taiko get sent too far back

### DISCUSSION THREAD

**Discussion by ethangrieve** [ppy/osu#30498] [Ideas]
the hit notes in taiko on lazer goes more just straight back then how it did before in stable where it slowly fills the HP bar and it just looks super weird now playing lazer compared to stable would appreciate at least an option to toggle this on/off so you also have the option to use the old version you can see the difference in these 2 clips. https://medal.tv/games/osu/clips/iZqlnke3nb2Kd-_Lk?invite=cr-MSxGVTQsNTcxNTY0NCw https://medal.tv/games/osu/clips/j0QDRZcftnLR2enZq?invite=cr-MSxQMmwsNTcxNTY0NCw both using the exact same skin 
![{EE28FD6D-6E2F-4928-A1A0-602C65D5F594}](https://github.com/user-attachments/assets/736d541d-cd1c-4bb9-ad52-e7e4ecb8dfad)
![{0B6DBF0D-A938-4AA3-98AB-B442A08EFF02}](https://github.com/user-attachments/assets/bfeddec4-131a-4e61-aa97-79ab3f946001)

**Comment by ethangrieve** [comment]
And yes i realise making it "track the healthbar" isnt realy feasable but you could just make the notes still go up to a similar spot to somewhat mimick the behavour so it doesnt look so off

**Reply by peppy** [reply]
I think it's definitely feasible, and maybe we can leave it on as the default or fixed behaviour until someone complains.

**Reply by velamyGH** [reply]
would really like to see this return as well as at least one other person who i couldnt get to complain on github lol
